.gitignore

```
@@ -1,7 +1,12 @@
 *Backup*
-STM32Cube/Debug/*
-STM32Cube/Release/*
+Debug/*
+Release/*
 **/.metadata
 **/.settings
 **/.vscode
 *.orig
+*.elf
+*.o
+
+.settings
```

`**/foo` is equivalent to `foo` (https://git-scm.com/docs/gitignore#_pattern_format) so having `.settings` twice is redundant
```suggestion
.metadata
.settings
.vscode
*.orig
*.elf
*.o
```

---

good catch, this was copypasted from last year rip